### DISCUSSION TITLE
Plans for future

### DISCUSSION THREAD

**Discussion by nikita-moor** [ilius/ayandict#4] [Q&A] [answered]
Hello @ilius. I have checked your app, and find it very interesting. There are very few analogs to StarDict or GoldenDict, and for many years computer dictionaries were decaying. That's why it's so exciting to see a new player on this field.

Could you share your plans for ayandict? Would it be a StarDict browser, or you plan to support multiple formats? Do you accept feature-requests or bug issues? Which features of StarDict format you plan to implement?

**Comment by ilius** [comment, marked as answer]
> Could you share your plans for ayandict? Would it be a StarDict browser, or you plan to support multiple formats?

I have implemented a custom SQLite format to improve startup time (loading large StarDict indexes can take a while), but fuzzy search with it is slow, so I haven't documented it yet. It's a work in progress.

I don't think I'm going to add any other format. Because we can just convert other formats easier with Python and pyglossary. And avoid bloating the dictionary.

I want to add tray icon and scan popup. Also maybe an http server with a web interface in the future if I find the time.

> Do you accept feature-requests or bug issues?

Of course you should report bugs, as issues.

But about features, I'm not very keen on adding too many features. You are still welcome to add a discussion page for it.


> Which features of StarDict format you plan to implement?

Ideally full support. But reading existing StarDict glossaries is priority. And I think that's done. If there is a glossary that does not work, let me know.

**Reply by ilius** [reply]
Edited.

**Reply by ilius** [reply]
I have added a web server, and web interface (with Brython) similar to Aard 2 web (I copied the css from there!)

**Comment by ilius** [comment]
Branch `v3` comes with lots of features and improvements

https://github.com/ilius/ayandict/tree/v3

If you use Linux or Mac, I can give you a binary (need homebrew for Mac for now).
